mypy/argmap.py

```diff
@@ -193,3 +193,21 @@ def expand_actual_type(self,
         else:
             # No translation for other kinds -- 1:1 mapping.
             return actual_type
+
+
+def expand_formal_type(formal_type: Type, actual_name: Optional[str], tuple_idx: int) -> Type:
+    if isinstance(formal_type, Instance) and formal_type.type.fullname() == 'mypy_extensions.Expand':
```

Comment:
This is ugly. How do I get something like `TypedDictType` (`ExpandType`) and make mypy map `mypy_extensions.Expand` to `mypy.types.ExpandType`?